### PR TITLE
feat(OSM): improve logs and remove some duplicate / unused code

### PR DESCRIPTION
**Pull request by bobeal**
Nothing spectacular : just improved some logs for better readability and removed some duplicate or unused code. But expecting more useful to come :)

### CODE REVIEW

File: adapters/adapter.py [hunk 1]
```diff
@@ -485,16 +485,9 @@ def uploadOSMService(self,request):
             LOG.debug(token)
             #file_to_upload = content['service']
             file_to_upload = request
-            file_composed = "@" + file_to_upload
```
Comment (felipevicens) [inline adapters/adapter.py:455]: @bobeal Are you sure this is not used?

File: adapters/adapter.py [hunk 2]
```diff
@@ -485,16 +485,9 @@ def uploadOSMService(self,request):
...
             file = {'nsd-create': open(file_to_upload, 'rb')}           
             data = {'service':file_to_upload}
 
-            HEADERS = {
```
Comment (felipevicens) [inline adapters/adapter.py:459]: @bobeal Are you sure this is not used?

**Review by felipevicens — COMMENTED** [review]
Just a double check 2 variables that are not used

File: adapters/adapter.py (hunk 1 above)
Comment (bobeal) [reply adapters/adapter.py:455]: yes, when uploading the NSD, only the `file_to_upload` variable is used.

File: adapters/adapter.py (hunk 2 above)
Comment (bobeal) [reply adapters/adapter.py:459]: yes, headers are passed directly in the curl command that is run some lines below.

**Pull request closed without merging**